src/main/webapp/js/adminHome.js

```diff
@@ -145,12 +180,12 @@ function addInstructorFromSecondFormByAjax() {
     
     var instructorDetails = $('#instructorName').val() + '|' + $('#instructorEmail').val()
                             + '|' + $('#instructorInstitution').val();
-    instructorDetailsList = [instructorDetails];
+    instructorDetailsList = [encodeInstructorDetails(instructorDetails)];
     var params = 'instructorshortname=' + $('#instructorShortName').val()
```

Comment:
Just had a look, `$.param` looks like it actually encodes the string properly, so all the separate encoding functions are unnecessary. I'll modify that part to use a query object then.